.copier-answers.yml

```diff
@@ -1,13 +1,13 @@
 # Changes here will be overwritten by Copier; NEVER EDIT MANUALLY
-_commit: 91ffffb
+_commit: 5c4fd02
 _src_path: gh:scipp/copier_template
 description: Transform directed acyclic graphs using map-reduce and groupby operations
-max_python: '3.12'
+max_python: '3.13'
 min_python: '3.10'
 namespace_package: ''
-nightly_deps: networkx/networkx
```

Comment:
Why is `networkx` removed from the nightly dependency?

Reply:
The current dev version of networkx drops python 3.10 which means we need to add a upper bound if cyclebane doesn't drop python 3.10 and with an upper bound we can't really test a nightly.